packages/block-editor/src/components/block-removal-warning-modal/index.js

```diff
@@ -65,7 +52,25 @@ export function BlockRemovalWarningModal( { rules } ) {
 			onRequestClose={ clearBlockRemovalPrompt }
 			size="medium"
 		>
-			<p>{ message }</p>
+			<div>
+				{ _n(
+					'Post or page content may not be displayed correctly if you delete this block:',
+					'Post or page content may not be displayed correctly if you delete these blocks:',
+					blockNamesForPrompt.length
+				) }
```

Comment:
Since this PR is aiming to remove the dependency on language that the block editor package shouldn't know about, would it be worth making this a label param and have the consumer pass it in, rather than baking it into the component? Just wondering if `template` is too specific for the block editor package, then is post or page content similarly too WP specific? (I don't have a strong opinion here, the thought just occurred to me while reading through the change 🙂)

Reply:
see [this comment](https://github.com/WordPress/gutenberg/pull/58952#issuecomment-1940340129)